build-farm/platform-specific-configurations/alpine-linux.sh

```diff
@@ -40,10 +40,12 @@ fi
 if [ "$(pwd | wc -c)" -gt 83 ]; then
   # Use /tmp for alpine in preference to $HOME as Alpine fails gpg operation if PWD > 83 characters
   # Alpine also cannot create ~/.gpg-temp within a docker context
-  GNUPGHOME="$(mktemp --directory /tmp/.gpg-temp.XXXXX)"
+  GNUPGHOME="$(mktemp -d /tmp/.gpg-temp.XXXXXX)"
 else
   GNUPGHOME="${WORKSPACE:-$PWD}/.gpg-temp"
+  mkdir -p "$GNUPGHOME"
```

Comment:
Noting that the original logic I had for this in https://github.com/adoptium/temurin-build/blob/07bda2837d9b4bc29235aa68283a7dcfc59a2626/build-farm/platform-specific-configurations/linux.sh#L139 was
```
if [ ! -d "$GNUPGHOME" ]; then
    mkdir -m 700 "$GNUPGHOME"
fi
```
which I think is a bit nicer than the separate `chmod` and doesn't need the `-p` but I'm ok either way.

Reply:
yep, i'm good with that, updated